src/Microsoft.Kusto.ServiceLayer/DataSource/KustoDataSource.cs

```diff
@@ -189,10 +202,10 @@ private void SetDatabaseMetadata(bool includeSizeDetails)
                         ClusterName = this.ClusterName,
                         MetadataType = DataSourceMetadataType.Database,
                         MetadataTypeName = DataSourceMetadataType.Database.ToString(),
-                        SizeInMB = includeSizeDetails == true ? row["sum_OriginalSize"].ToString() : null,
+                        SizeInMB = includeSizeDetails ? row["sum_OriginalSize"].ToString() : "",
```

Comment:
Changed to empty string so this doesn't throw a null reference exception when it tries to parse later.